InvenTree/InvenTree/settings.py

```diff
@@ -1177,3 +1162,23 @@
 # Magic login django-sesame
 SESAME_MAX_AGE = 300
 LOGIN_REDIRECT_URL = '/api/auth/login-redirect/'
+
+# Configuratino for API schema generation
+SPECTACULAR_SETTINGS = {
+    'TITLE': 'InvenTree API',
+    'DESCRIPTION': 'API for InvenTree - the intuitive open source inventory management system',
+    'LICENSE': {
+        'name': 'MIT',
+        'url': 'https://github.com/inventree/InvenTree/blob/master/LICENSE',
+    },
+    'EXTERNAL_DOCS': {
+        'description': 'More information about InvenTree in the official docs',
+        'url': 'https://docs.inventree.org',
+    },
+    'VERSION': str(inventreeApiVersion()),
+    'SERVE_INCLUDE_SCHEMA': False,
+    'SCHEMA_PATH_PREFIX': '/api/',
+}
+
+if SITE_URL:
+    SPECTACULAR_SETTINGS['SERVERS'] = [SITE_URL]
```

Comment:
The url subkey is missing here:

> # Optional list of servers.
>     # Each entry MUST contain "url", MAY contain "description", "variables"
>     # e.g. [{'url': 'https://example.com/v1', 'description': 'Text'}, ...]
>     'SERVERS': [],

Reply:
Maybe we can also use this then to auto generate a ts client (fully
Typesafe) for PUI (and also publish as npm package for the users) and a python client to eventually replace the InvenTree-Python package to reduce maintainance tasks there.

Reply:
The stuff needed for that is in https://github.com/inventree/InvenTree/pull/6440

Reply:
> The url subkey is missing here:
> 
> > # Optional list of servers.
> > ```
> > # Each entry MUST contain "url", MAY contain "description", "variables"
> > # e.g. [{'url': 'https://example.com/v1', 'description': 'Text'}, ...]
> > 'SERVERS': [],
> > ```
> 
> Apart from this it is working fine: I had been able to generate code with openapi-codegen without any hacking now!

Thanks, should be fixed now :)